extension-workspace-references.md

```diff
@@ -0,0 +1,115 @@
+# workspace/xreferences extension to LSP
+
+The `workspace/xreferences` extension to the Language Server Protocol (LSP) enables a language server to export all of the references to dependencies that code inside of the workspace makes.
```

Comment:
simplify sentence:

`... to export all of the references made from the workspace's code to its dependencies.`